src/App.js

```diff
@@ -13,11 +13,16 @@ class App extends Component {
     this.state = {
       movies: [],
       error: '',
-      user:{}
+      user:{},
+      isLoggedIn: false,
```

Comment:
I'll try to look into this more and see if we can't get it going before tomorrow